clientupdate/clientupdate_not_downloads.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build !((linux && !android) || windows)
```

Comment:
I didn't feel like reducing this with DeMorgan's etc. Reducing it is just error prone maintenance. Now that we have fancy new `go:build` syntax, it's easier to just take the exact same expression as the other file and wrap it in parens and slap a not on it.